apiserver/firewaller/firewaller.go

```diff
@@ -56,8 +56,8 @@ func NewFirewallerAPI(
 	accessUnit := common.AuthFuncForTagKind(names.UnitTagKind)
 	accessService := common.AuthFuncForTagKind(names.ApplicationTagKind)
 	accessMachine := common.AuthFuncForTagKind(names.MachineTagKind)
-	accessUnitOrService := common.AuthEither(accessUnit, accessService)
-	accessUnitServiceOrMachine := common.AuthEither(accessUnitOrService, accessMachine)
+	accessUnitOrService := common.AuthAny(accessUnit, accessService)
+	accessUnitServiceOrMachine := common.AuthAny(accessUnitOrService, accessMachine)
```

Comment:
Done - it turned out that this was actually doing Either(Either(unit, application), machine), so I changed it to Any(unit, application, machine) instead.